docs/decisions/app/0006-use-nextjs.md

```diff
@@ -0,0 +1,101 @@
+# Use Next.js for Template
+
+* Status: draft <!-- optional -->
+* Deciders: [list everyone involved in the decision] <!-- optional -->
+* Date: 2023-07-14 <!-- optional -->
+
+Technical Story: Add an ADR about why Next.js was selected: [ticket #166](https://github.com/navapbc/template-application-nextjs/issues/166) <!-- optional -->
+
+## Context and Problem Statement
+
+There are many React frameworks for full-stack and front-end TypeScript/JavaScript applications. This ADR reviews several options and the pros and cons of each for the template repo.
+
+## Decision Drivers <!-- RECOMMENDED -->
```

Comment:
can you remove what seems to be placeholders from ADR template, e.g. the `<!-- RECOMMENDED -->` and `<!-- REQUIRED -->` blocks